front/app/hooks/useEvents.ts

```diff
@@ -27,19 +26,20 @@ const sortEvents = (_events: IEventData[], sort: sort) => {
     : events.sort((a, b) => (getStart(b) < getStart(a) ? 1 : -1));
 };
 
+export type TEvents = IEventData[] | null | Error;
 export default function useEvents(parameters: InputParameters) {
-  const [events, setEvents] = useState<IEventData[] | undefined | null | Error>(
-    undefined
-  );
-  const [projectIds, setProjectIds] = useState<string[]>(
-    parameters.projectIds ?? []
-  );
-  const [currentPage, setCurrentPage] = useState<number>(
-    parameters.currentPage ?? 1
-  );
+  const [events, setEvents] = useState<TEvents>(null);
+  const [projectIds, setProjectIds] = useState<string[] | null>(null);
+  const [currentPage, setCurrentPage] = useState(1);
   const [lastPage, setLastPage] = useState(1);
   const [pageSize] = useState(parameters.pageSize ?? DEFAULT_PAGE_SIZE);
 
+  useEffect(() => {
+    if (parameters.projectIds !== undefined) {
+      setProjectIds(parameters.projectIds);
+    }
+  }, [JSON.stringify(parameters.projectIds)]);
+
```

Comment:
Project ids weren't updated. Not discovered until now because it's the first time need to wait for requests to get projectIds first.